### DISCUSSION TITLE
I have a question about distributed learning on pytorch geometric.

### DISCUSSION THREAD

**Discussion by Cow-Kite** [pyg-team/pytorch_geometric#9192] [Q&A] [answered]
I'm currently trying out the new package distributed in pytorch geometric.
I am running the /examples/distributed/pyg example posted on github.
I have a question. Can I only create one trainer process? Or can I create multiple trainers on one node to further accelerate distributed learning?
thank you.

**Comment by rusty1s** [comment, marked as answer]
Do you mean you want to run multiple processes of `DistributedDataParallel` on the same node? This is generally possible by simple executing the same script on the same node twice. Does that answer your question?

**Reply by Cow-Kite** [reply]
Thank you very much!
